src/Infrastructure/LeanCode.Kratos/KratosAuthenticationHandler.cs

```diff
@@ -76,49 +76,49 @@ protected override async Task<AuthenticateResult> HandleAuthenticateAsync()
         }
     }
 
-    protected virtual Task<KratosSession?> GetSessionAsync()
+    protected virtual async Task<IToSessionApiResponse?> GetSessionAsync()
     {
         if (Request.Headers.TryGetValue("X-Session-Token", out var token) && !string.IsNullOrEmpty(token))
         {
-            return api.ToSessionAsync(xSessionToken: token);
+            return await api.ToSessionAsync(xSessionToken: token.ToString());
```

Comment:
Before, this was a simple `string` parameter so implicit conversion from `StringValues` worked. Now it's `Option<string>` so it's not so easy anymore.